code/app/frontend/src/components/Answer/Answer.tsx

```diff
@@ -57,15 +59,16 @@ export const Answer = ({
         const handleCopy = () => {
             alert("Please consider where you paste this content.");
         };
-        document.addEventListener("copy", handleCopy);
+        var messageBox = document.getElementById("message-" + index);
```

Comment:
Maybe define `"message-" + index` as a variable then use it in both places

Also can use a template string